### PR TITLE
Refactor/remove wrong validation condition

### PR DESCRIPTION
**Pull request by slavistan**

### CODE REVIEW

File: src/useCases/consumption/attributes/NotifyPeerAboutIdentityAttributeSuccession.ts [hunk 1]
```diff
@@ -67,7 +64,7 @@ export class NotifyPeerAboutIdentityAttributeSuccessionUseCase extends UseCase<
         const candidatePredecessorsQuery: any = {
             "shareInfo.sourceAttribute": { $in: repositoryAttributeVersionIds },
             "shareInfo.peer": request.peer,
-            succededBy: { $exists: false }
+            succeededBy: { $exists: false }
```
Comment (slavistan) [inline src/useCases/consumption/attributes/NotifyPeerAboutIdentityAttributeSuccession.ts:67]: image

File: test/consumption/attributes-e2e.test.ts [hunk 2]
```diff
@@ -751,6 +751,23 @@ describe(NotifyPeerAboutIdentityAttributeSuccessionUseCase.name, () => {
...
+                },
+                tags: ["Champions"]
+            }
+        };
+        const { successor: sRASuccessor2 } = (await services1.consumption.attributes.succeedIdentityAttribute(succeedIdentityAttributeRequest2)).value;
+        const result2 = await services1.consumption.attributes.notifyPeerAboutIdentityAttributeSuccession({
+            attributeId: sRASuccessor2.id,
+            peer: services2.address
+        });
+        expect(result2.isSuccess).toBe(true);
```
Comment (Milena-Czierlinski) [inline test/consumption/attributes-e2e.test.ts:770]: Maybe check the content here, too.

**Review by Milena-Czierlinski — COMMENTED** [review]
Was there a discussion to change the versions we want to allow Identities to share with peers/notify peers about?

File: src/useCases/consumption/attributes/NotifyPeerAboutIdentityAttributeSuccession.ts (hunk 1 above)
Comment (jkoenig134) [reply src/useCases/consumption/attributes/NotifyPeerAboutIdentityAttributeSuccession.ts:67]: ouch

**Review by jkoenig134 — APPROVED** [review]

File: test/consumption/attributes-e2e.test.ts (hunk 2 above)
Comment (slavistan) [reply test/consumption/attributes-e2e.test.ts:770]: I (at least I think it was me) wrongfully implemented a policy that only allowed to send a succession-notification for the most up-to-date version of an identity attribute. We had a brief discussion this morning and unanimously agreed this to be arbitrary and unnecessary.

Please correct me if you have a different view on this :-)

Comment (Milena-Czierlinski) [reply test/consumption/attributes-e2e.test.ts:770]: That's fine with me.

**Pull request merged**